app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -15,35 +18,86 @@ $image-overlay-form-width: 350px;
   max-width: 100%;
 }
 
+.alchemy-image-overlay-dialog {
+
+  &.hide-form {
+
+    .picture-details-overlay {
+      right: -$image-overlay-form-width;
+    }
+
+    .zoomed-picture-background {
+      padding-right: 2*$default-padding;
+    }
+
+    .alchemy-image-overlay-close {
+      right: $picture-overlay-handle-width + 2*$default-padding;
+    }
+
+    .next-picture {
+      right: $picture-overlay-handle-width;
+    }
+
+    .spinner {
+      margin-left: 0;
+    }
+
+    .picture-overlay-handle {
+      right: 0;
+
+      .icon-angle-double-right {
+        @include transform(rotate(180deg) translate(50%, 50%));
+      }
+    }
+  }
+}
+
 .alchemy-image-overlay-body {
   width: 100%;
   height: 100%;
+
+  > .message {
+    margin: 2*$default-margin;
+    max-width: 400px;
+  }
 }
 
 .alchemy-image-overlay-close {
   width: 32px;
   height: 32px;
   top: 2*$default-padding;
-  right: $image-overlay-form-width + 2*$default-padding;
+  right: 2*$default-padding + $picture-overlay-handle-width + $image-overlay-form-width;
```

Comment:
`2*$default-padding` should be written with a single space on each side of the operator: `2 * $default-padding`